packages/frontend/src/lib/Link.svelte

```diff
@@ -19,6 +24,6 @@ async function click(): Promise<void> {
 }
 </script>
 
-<Link title={title} aria-label={$$props['aria-label']} on:click={click}>
-  <slot />
+<Link title={title} on:click={click}>
```

Comment:
question: where the ariaLabel property went ?

Reply:
See description - it was totally unused, so I figured there was no point keeping or porting it and easy enough to add back later. If you'd like, I can keep it.

Reply:
yes, it's only that I prefer to have cleanup being a separate operation of migrating a component

cleanup = removing
migrate = translate but no addition/removal